src/main/java/frc/robot/commands/auto/DriveAutoFieldOriented.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands.auto;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drive;
+
+public class DriveAutoFieldOriented extends Command {
+  private Drive drive;
+  private Translation2d destination;
+  private double rotation;
+
+  /** Creates a new DriveAuto. */
+  public DriveAutoFieldOriented(Translation2d destination, double rotation) {
+    this.drive = RobotContainer.drive;
+    addRequirements(this.drive);
+    this.destination = destination;
+    this.rotation = rotation;
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    drive.driveFieldOriented(destination, rotation);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
```

Comment:
Call `drive.stop()` here, because otherwise the robot will keep frickin' driving after this command ends.